orb.yml.tmpl

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+
+description: "Automatically builds and pushes Docker images."
+display:
+  source_url: "https://github.com/remind101/docker-build"
+
+jobs:
+  # build a docker image and push to docker hub and ECR
+  build-push:
+    docker:
+      - image: "remind101/docker-build:{{version}}"
```

Comment:
This is not how I thought this would work. I thought we wouldn't use docker in the orb itself. This seems fine, and once we use this orb everywhere we can change it whenever we want, but I think it'd be nice to get rid of the bootstrapping problem.

Reply:
My first priority is getting everything pushing to ECR, so I'm doing the minimal thing here just to get everything transitioned to the orb with ECR push. We can follow up later with an update to remove the dependence on docker hub if desired. It's more complicated that just embedding the script, because
- The script depends on having the AWS CLI which we need to provide somehow
- Our builds still rely on using docker hub as a layer cache